admin/package-files/flocker-zfs-agent.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Flocker ZFS Agent
+
+[Service]
+ExecStart=/usr/bin/flocker-zfs-agent $FLOCKER_NODE_NAME $FLOCKER_CONTROL_NODE
```

Comment:
So docs will say "edit `/etc/sysconfig/flocker-zfs-agent` and set these options"? I suggest maybe calling it (and other related files correspondingly) `/etc/sysconfig/flocker-agent`, we're probably going to replace the ZFS-specific agent with a generic agent that figures out if it should do ZFS or EBS or what.